### ISSUE TITLE
Code review: explaining options for session

### ISSUE THREAD

**Issue by nicola** [linkeddata/ldnode#39]
I was wondering why every single option would be needed

```
 app.use(session({
    secret: 'node-ldp',
    saveUninitialized: false,
    resave: false
  }));
```

I consider the `secret: 'node-ldp'` by default a serious security issue

**Comment by martinmr** [comment]
I set saveUninitialized and resave to false based on the express-session documentation: https://github.com/expressjs/session. 

I agree that setting secret to a fixed string is a problem. I will fix it and submimt a PR.

**Comment by nicola** [comment]
No! Please don't! Let's discuss the secret on friday

**Comment by martinmr** [comment]
I already submitted a PR if you want to have a look.

**Comment by nicola** [comment]
Perfect I have seen it, it's great, although I have some questions about that that we should discuss

**Comment by martinmr** [comment]
Clossing as issue was already fixed by #77.
